src/main/java/laddergame/domain/gameelements/Player.java

```diff
@@ -0,0 +1,36 @@
+package laddergame.domain.gameelements;
+
+public class Player {
+    private static final Name RESERVED_NAME = new Name("all");
```

Comment:
```suggestion
    private static final Name RESERVED_NAME = Name.all();
```

이럴 때 정적 팩터리 메서드를 사용하는걸 추천합니다.
문자열을 객체 외부가 아닌 내부에서 관리해야 수정 범위를 줄일 수 있어요

Reply:
반영해주었습니다!!